feature/about/src/main/java/io/github/droidkaigi/confsched2023/about/AboutScreen.kt

```diff
@@ -81,8 +90,29 @@ private fun AboutScreen(
     versionName: String?,
     onLinkClick: (url: String) -> Unit,
 ) {
+    val scrollBehavior = TopAppBarDefaults.pinnedScrollBehavior()
     Scaffold(
         modifier = Modifier.testTag(AboutScreenTestTag),
+        topBar = {
+            TopAppBar(
+                title = {
+                    if (scrollBehavior.state.overlappedFraction == 0f) {
+                        Text(
+                            text = AboutStrings.Title.asString(),
+                            style = MaterialTheme.typography.headlineLarge,
+                            fontWeight = FontWeight.Medium,
+                        )
+                    } else {
+                        Text(
+                            text = AboutStrings.Title.asString(),
+                            color = Color.Unspecified.copy(alpha = scrollBehavior.state.overlappedFraction),
```

Comment:
When scrolled, the title is made smaller and gradually displayed according to the scroll position.
I wanted to use LargeTopAppBar if possible, but I couldn't create a good layout on the About screen because the icon part at the top was in the way.
Therefore, TopAppBar is used to simulate the behavior of LargeTopAppBar by controlling the transparency.